apis/Google.Cloud.Metadata.V1/Google.Cloud.Metadata.V1/project.json

```diff
@@ -0,0 +1,43 @@
+﻿{
+  "version": "0.1.0-*",
+  "description": "Google Compute Engine Metadata v1 client library",
```

Comment:
Sorry, I'm not clear on the first part of your comment. Are you saying this should use a different format for the description? As for the v1 part, I thought about this some more and realized it probably makes sense to keep it because the metadata URL is versioned (see "Current version: v1" here: https://cloud.google.com/compute/docs/storing-retrieving-metadata). ".../computeMetadata/**v1**/..." is also part of all URLs for accessing this metadata